### DISCUSSION TITLE
Tweeting GitHub Repo (displays old description)

### DISCUSSION THREAD

**Discussion by CleasbyCode** [community/community#48572] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

When tweeting a repository, it will display an old description of the repo. 

How long does it take for the updated repo description to appear when sharing via Twitter.

Thanks,
Nick.

**Comment by valentinojosh** [comment]
Hi Nick!
If after a couple days it doesnt change, or if you just want to get it changed ASAP you have two main options:

1. Twitter Card Validator: https://cards-dev.twitter.com/validator
This would make Twitter update the information cached about the link. 

2. Change the URL: If you change the url to the repo it would force the change as well.

Hope this helps!
Best,
Josh V

**Comment by CleasbyCode** [comment, marked as answer]
Thanks.
